Common/Crypto/VaultItemCryptor.swift

```diff
@@ -0,0 +1,30 @@
+import CryptoKit
+import Foundation
+
+struct VaultItemCryptor {
+    
+    let masterKey: SymmetricKey
+    
+    func encode(_ vaultItem: VaultItem) throws -> Data {
+        let encodedInfo = try VaultItemInfoEncode(vaultItem.info)
```

Comment:
could this just be info? since it is in the `encode` method it seems redundant to call it `encodedInfo`

Reply:
I think encodedInfo is better since it describes the fact that it is encoded (which is important)

Reply:
encodedInfo renamed to  encodedVaultItemInfo to match decode function naming